{{ cookiecutter.github_repo_name }}/tests/test_calculator.py

```diff
@@ -0,0 +1,17 @@
+from {{cookiecutter.package_dir_name}} import calculator
+
+
+def test_dot_product_2D():
+    """Test the dot product function in 2D."""
+    a = [1, 2]
+    b = [3, 4]
+    expected = 11
+    assert calculator.dot_product(a, b) == expected
```

Comment:
let's use the `actual`/`expected` syntax here, so add an extra line that is `actual = calculator.dot_product(a, b)` then `assert actual == expected`